ansible/plan.sh

```diff
@@ -32,17 +32,17 @@ do_prepare() {
   mkdir -p "${pkg_prefix}/lib/python2.7/site-packages"
   mkdir -p "${pkg_prefix}/share"
   mkdir -p "${pkg_prefix}/etc"
+  pip install -r requirements.txt
+  pip install packaging
 }
 
 do_build() {
-  python setup.py build
+  make
 }
 
 do_install() {
-  python setup.py install \
-    --prefix="${pkg_prefix}" \
-    --optimize=1 \
-    --skip-build
+  make install  
```

Comment:
`make install` unfortunately won't respect `$pkg_prefix`, leaving `$pkg_prefix/lib/python2.7/` empty.  Looking at the Makefile, we should be able to use `PREFIX=$pkg_prefix make install`, but that was installing the packages into `core/python2/` 😱 , which is not what we want.   I tried moving back to the `python setup.py` iteration and it seems to have worked.

Reply:
Changed back to setup